src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -1662,7 +1662,7 @@ public function testTrustedProxies()
         // trusted proxy via setTrustedProxies()
         Request::setTrustedProxies(array('3.3.3.3', '2.2.2.2'));
         $this->assertEquals('1.1.1.1', $request->getClientIp());
-        $this->assertEquals('real.example.com', $request->getHost());
+        $this->assertEquals('foo.example.com', $request->getHost());
```

Comment:
I think nobody reported this bug because nobody is doing double reverse proxies (which is what this needs to be hit). But I doubt this will break, since having a mixed "port from edge" + "host from the second proxy" already makes little sense.